src/platforms/atomic-kms/server/gbm_display_allocator.cpp

```diff
@@ -115,9 +121,14 @@ class GBMBoFramebuffer : public mg::FBHandle
 
 namespace
 {
-auto create_gbm_surface(gbm_device* gbm, geom::Size size, mg::DRMFormat format, std::span<uint64_t> modifiers)
-    -> std::shared_ptr<gbm_surface>
+auto create_gbm_surface(
+    gbm_device* gbm,
+    geom::Size size,
+    mg::DRMFormat format,
+    std::span<uint64_t> modifiers,
+    std::shared_ptr<mga::GbmQuirks> runtime_quirks) -> std::shared_ptr<gbm_surface>
```

Comment:
`create_gbm_surface()` does not need ownership of `runtime_quirks` [sic].

```suggestion
    mga::GbmQuirks const& gbm_quirks) -> std::shared_ptr<gbm_surface>
```